cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -46,6 +46,7 @@
   // Default configs
   private static final String DEFAULT_METRIC_REPORTER_SAMPLER_GROUP_ID = "CruiseControlMetricsReporterSampler";
   private static final long DEFAULT_RECONNECT_BACKOFF_MS = 50L;
+  private static final long ACCEPTABLE_NETWORK_DELAY_MS = 100L;
```

Comment:
Is this number based on observed data?

Reply:
No, but I feel 100ms is a quite conservative value for one-way network latency.
This maybe a good reference, it is latency between regions in AWS.
https://www.cloudping.co/
I expect in normal production environment, CC should be sitting in the same datacenter with Kafka cluster so the latency should be sub millisecond.